app/livechat/server/lib/QueueManager.js

```diff
@@ -19,7 +18,7 @@ export const QueueManager = {
 			department: Match.Maybe(String),
 		}));
 
-		if (!Livechat.online(guest.department)) {
+		if (!checkServiceStatus({ guest, agent })) {
```

Comment:
is this the only place that needs to be changed? or should all places that call `Livechat.online` be changed to call `checkServiceStatus` instead?

Reply:
Yes, I got your point, I thought the same when I made this change.
The fact is we can't replace all of the calls for `Livechat.online` with  `checkServiceStatus` because when the first method checks the service status, we don't have the extra info, such as `guest`, `agent`, etc. We only have that information when a new Omnichannel room is requested, but all requests for a new room will pass through that part of the code, though.